api/controllers/models.py

```diff
@@ -211,10 +211,9 @@ def get_model_detail(credentials, mid):
 
         if not is_current_user and query_result[0].is_anonymous:
             model["username"] = "anonymous"
-            model["user_id"] = -1
```

Comment:
I realised that originally both uid and this additional `user_id` were being passed to the frontend for some reason. This meant that when I obfuscated the `user_id` to `-1` one could still find out the actual model owner through `uid`.

This additional `user_id` attribute doesn't appear to serve any purpose that `uid` doesn't already so I've removed it